lib/senders/email.js

```diff
@@ -935,6 +935,7 @@ module.exports = function (log, config) {
           location: this._constructLocationString(message),
           passwordChangeLink: links.passwordChangeLink,
           passwordChangeLinkAttributes: links.passwordChangeLinkAttributes,
+          accountSettingsUrl: this.accountSettingsUrl,
```

Comment:
On line 914, you can update the helper function to create links with `this.accountSettingsUrl`.

```
var links = this._generateLinks(this.accountSettingsUrl, message.email, {}, templateName)
```

This will append correct service, utm params etc.

Finally, this line could be `link: links.link`.